.goreleaser.yml

```diff
@@ -54,7 +54,7 @@ checksum:
   name_template: "checksums.txt"
 
 changelog:
-  use: github-native
+  skip: true
```

Comment:
i skipped this now that the release notes will actually need to be a combination of the this repo and the UI repo